.dockerignore

```diff
@@ -1,11 +1,15 @@
 # we want to get the git commit SHA but not all the binary repo data
 .git/objects/pack
 bin/
+**/*.pyc
```

Comment:
Forgot to mention this earlier... could you do a `git rm **/*.pyc` and see if anything is currently checked in which shouldn't be?  Could also be in a separate PR